### PR TITLE
Added files for Week 3 warmup 's 

### PR DESCRIPTION
**Pull request by eliz79**
Week 3 Warmups

### CODE REVIEW

**Comment by cheuschober** [comment]
retest this please

File: task_08.py [hunk 1]
```diff
@@ -18,3 +18,7 @@
 
 
 """
+
+NERVOUS_AS = str.strip(NERVOUS_AS)[10:61]
```
Comment (cheuschober) [inline task_08.py]: In this case, I wanted you to use the different forms of strip to strip from the right and left. You can watch this week's Q/A video for a sense of how to use those functions.

File: task_10.py [hunk 2]
```diff
@@ -3,3 +3,7 @@
 """Task 10"""
 
 MOVIE = 'dr. strangelove or: how i learned to stop worrying and love the bomb'
+
+ENTITLED = str.title(MOVIE)
```
Comment (cheuschober) [inline task_10.py]: since Movie is also a string, you don't have to call the `str` function, eg `MOVIE.title()` is just fine!

File: task_12.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Some numeric types."""
+
+import decimal
+import fractions
+
+INTVAL = decimal.Decimal('1')
```
Comment (cheuschober) [inline task_12.py]: So close there! `INTVAL = 1`

File: task_14.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+#! usr/bin/env python
+# -*- coding: utf-8 -*-
+"""This is a docstring"""
+
+IS_TRUE = True
+IS_FALSE = False
+IS_NONE = None
+
+INTEGER_EQUIV = IS_TRUE == True and (IS_FALSE) == False
```
Comment (cheuschober) [inline task_14.py]: The goal was to show how True and False have integer equivalents in 1 and 0.

File: task_08.py (hunk 1 above)
Comment (eliz79) [inline task_08.py]: Reviewed the video. A bit clearer.

**Comment by eliz79** [comment]
I retested Task_13.py and made a commit

**Pull request merged**